config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+    get '/books', to: "books#index"
+    get '/books/:id', to: "books#show"
```

Comment:
Change this.

The proper way to do this is using resources following this https://guides.rubyonrails.org/routing.html